tests/src/JIT/Directed/pinning/object-pin/mirror.cpp

```diff
@@ -2,6 +2,7 @@
 #define EXPORT_API extern "C" __declspec(dllexport)
 #else
 #define EXPORT_API extern "C" __attribute__((visibility("default")))
+#define __int32 int
```

Comment:
Hmm, the native parts of the tests were being built before your change just fine, so I am not sure I understand the changes here and in other native headers / files.